src/Azure.Functions.Cli/StaticResources/node-v4-templates.json

```diff
@@ -0,0 +1,424 @@
+﻿[
```

Comment:
Can you pull the latest templates from here please:
https://gist.github.com/ejizba/4780c79193acf5e0082a9bdb50bde4a3